lib/govuk_connect/cli.rb

```diff
@@ -932,23 +936,21 @@ def types
   def main(argv)
     check_ruby_version_greater_than(required_major: 2, required_minor: 0)
 
+    extra = []
```

Comment:
It took me a few readings of the commit message to untangle the three kinds of "extra" here, which indicates the code may be confusing on its own:

- `extra` means `extra_args_after_double_dash`
- `rest` means `extra_args_before_double_dash`
- `options` means `govuk_connect_options`

These are long names, but more understandable than the current ones. Could we use these instead of them?

Reply:
:+1: